server/data/types/locationsApi/location.d.ts

```diff
@@ -47,8 +47,8 @@ export declare interface Location {
   childLocations: string[]
   changeHistory: {
     attribute: string
-    oldValue: string
-    newValue: string
+    oldValues: string[]
+    newValues: string[]
```

Comment:
@Mjwillis intends to remove oldValue and newValue eventually, so I've removed them early and swapped over to the new attributes